### DISCUSSION TITLE
Autocomplete in Fore

### DISCUSSION THREAD

**Discussion by timathom** [Jinntec/Fore#213] [Q&A] [answered]
Is there an out-of-the-box (or recommended) approach to typeahead or autocompletion in Fore? Some old demos come up in search results here, but I don't see anything in the current demo suite.

**Comment by JoernT** [comment, marked as answer]
just pushed an example to dev branch

In
/demo/fx-control.html
under 'Datalist (autocomplete) - dynamic'

you find an example using native HTML datalist bound to a list of instance nodes. It uses the `data-ref` type of binding as with tables :)

**Reply by timathom** [reply]
Nice! Is it possible to prevent the pop-up with the list of values from displaying on focus in?

**Reply by JoernT** [reply]
Unfortunately there's not much to control with native datalist so i guess the answer is no. If you want that you'll have to go with one of the js solutions out there. If you'd like to attempt that i'd be interested in the findings.
